src/services/positionService.ts

```diff
@@ -713,6 +733,79 @@ export default class PositionService {
     );
   }
 
+  async buildXcallDepositParams(
+    fromToken: Token,
+    toToken: Token,
+    fromValue: string,
+    frequencyType: BigNumber,
+    frequencyValue: string,
+    destinantionChainID: number,
+    yieldFromPossible?: string,
+    yieldToPossible?: string
+  ) {
+    const destinantionUSDC = this.connextService.getNativeUSDCAddress(destinantionChainID);
```

Comment:
would this actually be destinationUSDC? or it should be the token I want to get on the destination chain?, btw typo on `destination`